### DISCUSSION TITLE
코로나 자가진단 앱 커뮤니티

### DISCUSSION THREAD

**Discussion by lhwdev** [lhwdev/covid-selftest-macro#1] [공지]
코로나 자가진단 앱 커뮤니티에 오신 것을 환영합니다!
안타깝게도 깃허브 시스템 상의 제약으로 여기에 글을 쓰려면 깃허브 계정이 있어야 합니다.
가끔 여기에 개발 근황(?) 같은 거나 버그 공지도 올라올 예정입니다. (물론 버그 공지는 앱 화면에서도 볼 수 있습니다)

**Comment by lkk1110123** [comment]
정말 잘 쓰고있습니다!
어떻게 만드는지 배우고싶어요

**Reply by lhwdev** [reply]
감사합니다!
만드는건 앱을 만들 줄 안다면, 크롬 등의 개발자도구에서 네트워크를 분석하거나 js 코드를 해석해서 앱으로 옮기면 끝입니다.
되게 쉬운거처럼 썼지만 그리 쉽진 않더라고요.. (이런걸 물어보신게 맞는지..)

**Reply by lkk1110123** [reply]
네 맞아요 근데 혹시 매크로인걸 티나지 않게 일정 시간 내에서 랜덤으로 자가검진 하는 기능을 추가해주실수 있으신가요??

**Reply by lhwdev** [reply]
안그래도 그 기능을 처음 만들 때부터 구상하고 있었는데 시간이 없어서 못만들고 있었네요. 곧 패치할게요.

**Reply by lhwdev** [reply]
.. 라고 했는데 시간이 없네요.
내부구조도 갈아엎어서 오래 걸리겠네요.
죄송합니다. (ㅠㅠ)
아무튼 저도 있으면 좋을거 같은 기능이니 언젠가는 패치하겠습니다.

**Reply by lkk1110123** [reply]
네 바쁘신데 죄송합니다...배너광고도 추가해보시는게 어떠세요??
